Source/missiles.cpp

```diff
@@ -4790,39 +4790,19 @@ void __fastcall MI_Weapexp(int i)
 
 void __fastcall MI_Misexp(int i)
 {
-	int v1;           // edi
-	int v2;           // esi
-	BOOLEAN v3;       // zf
-	int v4;           // ecx
-	int v5;           // eax
-	int ExpLight[10]; // [esp+8h] [ebp-28h]
+	int ExpLight[10]={9,10,11,12,11,10,8,6,4,2}; // [esp+8h] [ebp-28h]

```

Comment:
Consider removing the comment.

Also, do you have `clang-format` installed? It should be able to automatically format the source code for you, so that e.g.

```c
 	int ExpLight[10]={9,10,11,12,11,10,8,6,4,2};
```

becomes

```c
 	int ExpLight[10] = {9, 10, 11, 12, 11, 10, 8, 6, 4, 2};
```

Reply:
yeah I do, sometimes it doesn't activate, no idea why, do I have to press enter at the end of the line or something special?

Reply:
Good question. I don't know. I run it from the terminal :D